docs/bouwblokken/data/index.md

```diff
@@ -6,7 +6,13 @@ icon: material/database
 
 ![data](../../afbeeldingen/bouwblokken/data.jpg "visuele weergave data"){align="right"; width="50%"}
 
-Hier komt een beschrijving van dit bouwblok. 
+Het ontwikkelen en gebruiken van algoritmes en AI-systemen kan niet gepaard gaan zonder het verwerken van data. 
+In het geval van AI wordt data gebruikt om het algoritme te trainen, te valideren en te testen. 
+Wanneer beslissingen worden genomen op basis van de output van een algoritme of AI-systeem, dan wordt deze beslissing ook genomen op basis van de onderliggende data. 
```

Comment:
Wanneer beslissing en besluiten worden genomen op basis van de output van een algoritme of AI-systeem, dan wordt dit ook gedaan op basis van de onderliggende data.

Reply:
opgelost in onderstaande commits